client/dashboard/data/index.ts

```diff
@@ -472,3 +478,19 @@ export const leaveSite = async ( siteIdOrSlug: string, userId: number ) => {
 		path: `/sites/${ siteIdOrSlug }/users/${ userId }/delete`,
 	} );
 };
+
+export const fetchP2HubP2s = async (
```

Comment:
nit: Renaming this to countP2HubP2s seems more descriptive.

Reply:
But the endpoint returns the P2s based on the given site ID of the p2 hub 🤔

Reply:
Yes. It returns below

```ts
type Response = {
  data: P2Site[];
  foundItems: number;
  totalItems: number;
}
```